src/oatpp/network/client/SimpleTCPConnectionProvider.cpp

```diff
@@ -75,7 +75,7 @@ std::shared_ptr<oatpp::data::stream::IOStream> SimpleTCPConnectionProvider::getC
   }
 
   struct addrinfo* currResult = result;
-  oatpp::v_io_handle clientHandle =- 1;
+  oatpp::v_io_handle clientHandle = oatpp::v_io_handle(-1);
```

Comment:
Let's use a new `INVALID_IO_HANDLE` constant here, from the previous comment.

Reply:
Sure